src/components/css/react-dates-overrides.css

```diff
@@ -0,0 +1,32 @@
+/*
+* From the react-dates docs:
+* "Note: This component assumes box-sizing: border-box is set globally in your page's CSS."
+* If box-sizing: border box isn't set globally, the picker appears a bit broken.
+*/
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
```

Comment:
This is a good idea in principle but it's a bit dangerous to set the wildcard selector. Other components might be expecting the default `content-box`; we should double check if the layout looks different. I'd prefer if the above lines (6-11) were more explicitly in a "global" or "base" CSS file with the above comment explaining the rationale.